hiding-in-the-clouds_secret-management-with-openstack-barbican/heat-templates/master/get_certs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Can you symlink this script from heat/templates/mitaka, please (it will do the job for both releases)?